### DISCUSSION TITLE
A few issues/ideas

### DISCUSSION THREAD

**Discussion by torevor** [vzhd1701/gridplayer#37] [Ideas]
- I have VLC configured such that the video pauses when minimized and resumes when restored - gridplayer pauses when minimized, but all videos must be manually replayed upon restore

- when a new video is added and starts a new row, it takes up the whole width and cuts off the top and bottom....this is kind of both a bug and a feature, maybe could be configurable?

- being prompted to save a playlist on close, would be nice to have the option to turn off

- I notice that loop configuration is saved in the playlist....nice.

AMAZING program, have wanted something like this for ages.

**Comment by andr-groth** [comment]
> being prompted to save a playlist on close, would be nice to have the option to turn off

Yes, please add this option.

**Comment by vzhd1701** [comment]
> I have VLC configured such that the video pauses when minimized and resumes when restored - gridplayer pauses when minimized, but all videos must be manually replayed upon restore

Done

> when a new video is added and starts a new row, it takes up the whole width and cuts off the top and bottom....this is kind of both a bug and a feature, maybe could be configurable?

You can configure grid size and alignment https://github.com/vzhd1701/gridplayer/issues/17#issuecomment-987655089

> being prompted to save a playlist on close, would be nice to have the option to turn off

Done. Now you have "Warn about unsaved changes" option in settings
